packages/api/src/repository/ResultRequest.ts

```diff
@@ -66,4 +73,19 @@ export class ResultRequestRepository {
     if (resultRequest?._id)
       return { ...resultRequest, id: resultRequest._id.toString() }
   }
+
+  private isValidResultRequest (
+    resultRequest: Omit<ResultRequestDbObject, '_id'>
+  ): boolean {
+    return !!(
+      resultRequest.address &&
+      resultRequest.drTxHash &&
+      resultRequest.feedId &&
+      resultRequest.feedId &&
+      resultRequest.label &&
+      resultRequest.requestId &&
+      resultRequest.result &&
+      resultRequest.timestamp
+    )
```

Comment:
Same suggestion as before (ok to ignore if you don't see the need)
```suggestion
    return !!Object.values(resultRequest).reduce(prev, val => prev && val)
```